sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -0,0 +1,273 @@
+<template>
+  <formio
+    :formName="selectedForm"
+    :data="initialData"
+    :readOnly="isReadOnly"
+    @loaded="formLoaded"
+    @changed="formChanged"
+    @submitted="submitted"
+    @customEvent="customEvent"
+  ></formio>
+  <StudentApplicationCommonActions
+    :isFirstPage="isFirstPage"
+    :isLastPage="isLastPage"
+    @wizardGoNext="wizardGoNext"
+    @wizardGoPrevious="wizardGoPrevious"
+  />
+</template>
+
+<script lang="ts">
+import formio from "@/components/generic/formio.vue";
+import StudentApplicationCommonActions from "@/components/common/StudentApplicationCommonActions.vue";
+import {
+  OfferingIntensity,
+  WizardNavigationEvent,
+  FormIOCustomEvent,
+} from "@/types";
+import { ref, SetupContext } from "vue";
+import {
+  useFormioComponentLoader,
+  useFormioDropdownLoader,
+  useFormioUtils,
+} from "@/composables";
+
+export default {
+  emits: [
+    "loadForm",
+    "submitApplication",
+    "customEventCallback",
+    "setFirstLastPage",
+  ],
+  components: {
+    formio,
+    StudentApplicationCommonActions,
+  },
+  props: {
+    initialData: {
+      type: Object,
+      required: true,
+    },
+    selectedForm: {
+      type: String,
+      required: true,
+    },
+    isReadOnly: {
+      type: Boolean,
+      required: true,
```

Comment:
isReadOnly is this required property ?

Reply:
yes, the value is passes from the parent component